src/Behat/Testwork/Call/Handler/RuntimeCallHandler.php

```diff
@@ -105,6 +105,7 @@ private function executeCall(Call $call)
         $return = $exception = null;
 
         try {
+            $arguments = array_values($arguments);
             $this->validator->validateArguments($reflection, $arguments);
             $return = $callable(...array_values($arguments));
```

Comment:
This `array_values` can be removed now that you moved it earlier (but is it actually an issue in `validateArguments` or are you using a Behat version which does not have this `array_values` yet ?)

Reply:
Good catch, I have removed duplicated call :)

I am not sure, where this issue comes from I could not track it all the way. 
However I am pretty sure, you will see the issue, when test suite will be set up for PHP 8.